app/channels/test/submission/test_runs_channel.rb

```diff
@@ -0,0 +1,14 @@
+module Test
+  module Submission
+    class TestRunsChannel < ApplicationCable::Channel
+      def subscribed
+        stream_for submission
+      end
+
+      private
+      def submission
+        @submission ||= ::Submission.find_by(uuid: params[:uuid])
```

Comment:
The `||=` pattern is used to ensure that the calculation is only done once, right? At other places we use Mandate's `memoize` support for that. It might be good to use the same pattern everywhere, or is there a benefit to this method over using Mandate?

Reply:
Ah yes! I always forget.